src/base/utils/fs.cpp

```diff
@@ -238,40 +239,10 @@ qulonglong Utils::Fs::freeDiskSpaceOnPath(const QString &path)
 {
```

Comment:
you should change the return type to `qint64`, same as `QStorageInfo::bytesAvailable`
And return `-1` for failures.